models/distill/__main__.py

```diff
@@ -0,0 +1,144 @@
+from models.bert.args import get_args
+from models.bert.model import BertForSequenceClassification
+from __future__ import absolute_import, division, print_function
+from pytorch_pretrained_bert.file_utils import PYTORCH_PRETRAINED_BERT_CACHE
+#from datasets.dataset4bert import DataProcessor, Sst2Processor, ReutersProcessor, AAPDProcessor, InputExample, InputFeatures
```

Comment:
Unnecessary comments.

Reply:
This file was removed in 0cdb70ac3322a7187ae5443b641fd44709aef60a